sentry/src/lib.rs

```diff
@@ -71,11 +82,16 @@ where
     }
 }
 
-async fn handle_routing(req: Request<Body>, adapter: impl Adapter) -> Response<Body> {
-    if req.uri().path().starts_with("/channel") {
-        crate::routes::channel::handle_channel_routes(req, adapter).await
-    } else {
-        Err(ResponseError::NotFound)
+async fn handle_routing(
+    req: Request<Body>,
+    (adapter, config): (impl Adapter, Config),
+) -> Response<Body> {
+    match (req.uri().path(), req.method()) {
+        ("/cfg", &Method::GET) => crate::routes::cfg::return_config(&config),
+        (route, _) if route.starts_with("/channel") => {
```

Comment:
Instead of having routes defined at multiple levels within the application, can we move all route definitions here? We can move the `/channel/list` & `/channel` route definitions here

Reply:
This change request is not part of this PR. Please consider first opening an issue to discuss how to do it and then to make the change instead of blocking this issue.